.github/workflows/CreateRelease.ps1

```diff
@@ -0,0 +1,95 @@
+# Required parameter specifying the secret access token to view releases and manage approvals.
+param([Parameter(Mandatory=$true)][string]$accessToken)
+
+# Function to help log out useful information.
+function log ([parameter(mandatory)][String]$msg) { Add-Content log.txt $msg }
+
+# Generate a log file.
+New-Item ./log.txt -ItemType File -Force
+
+# Releases are only created every other week even though the action occurs every week.
+# Identify if this is the second week of a sprint.
+$currentDate = Get-Date
+$currentDate = $currentDate.ToUniversalTime().Date
+$firstRun = ([datetime]::ParseExact('10-23-2020', 'MM-dd-yyyy', $null)).Date
+$weeksSince = [math]::floor(($currentDate - $firstRun).TotalDays / 7)
```

Comment:
I think the reason to not do a %14 is because to allow the schedule to change. This way, it could be a tuesday initially and then change to a wednesday and this script wouldn't have to change, just the schedule for the github action.

---
In reply to: [516969486](https://github.com/microsoft/dicom-server/pull/410#discussion_r516969486) [](ancestors = 516969486)